docker-compose.yml

```diff
@@ -112,7 +115,7 @@ services:
       interval: 30s
```

Comment:
Can we drop the interval and increase retries to match the other services?

Reply:
You mean to remove the interval in mysql and increase the retries to 5?

Reply:
Sorry, I meant drop it to ~10s and increase retries to 9 to match the timeout of 90s.

Reply:
changed.